util/topsql/stmtstats/observer.go

```diff
@@ -39,7 +39,7 @@ type StatementObserver interface {
 	// It may be called without a MySQL protocol when this session is created in background.
 	OnDigestKnown(sqlDigest []byte, planDigest []byte)
 
-	// The following events should be called during reading from wire protocol.
-	OnExecuteBegin()
-	OnExecuteFinish()
+	// The following events should be called during reading from internal protocol.
```

Comment:
There is no internal protocol.

```suggestion
	// The following events should be called when executing an internal statement. The execution may happen
	// directly over a session that is built from the wire connection, or an internal session in the pool.
```